### PR TITLE
Update release-faq.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor punctuation fixes.

### CODE REVIEW

File: source/process/release-faq.rst [hunk 1]
```diff
@@ -93,19 +93,19 @@ Frequently Asked Questions
...
 **Q: What changes were made to the dev release process to account for the rotating feature and quality releases? https://developers.mattermost.com/internal/release-process/**
 
 - A: PR with changes was merged `here <https://github.com/mattermost/mattermost-developer-documentation/pull/182>`__.
 
 **Q: What changes were made to the team release process to account for the rotating feature and quality releases? https://docs.mattermost.com/process/release-process.html**
 
-- A: Separate checklists for `Bug Fix release <https://docs.mattermost.com/process/bug-fix-release.html>`__ and `Feature release <https://docs.mattermost.com/process/feature-release.html>`__ were created.
+- A: Separate checklists for `Bug fix release <https://docs.mattermost.com/process/bug-fix-release.html>`__ and `Feature release <https://docs.mattermost.com/process/feature-release.html>`__ were created.
 
 **Q: How does Redux branching work?**
```
Comment (justinegeffen) [inline source/process/release-faq.rst]: This needs an answer.

File: source/process/release-faq.rst [hunk 2]
```diff
@@ -73,11 +73,11 @@ Frequently Asked Questions
  
 **Q: How is cherry-picking done?**
```
Comment (justinegeffen) [inline source/process/release-faq.rst]: Needs an answer.

Comment (amyblais) [reply source/process/release-faq.rst]: Can link to https://developers.mattermost.com/contribute/getting-started/branching/.

File: source/process/release-faq.rst [hunk 3]
```diff
@@ -73,11 +73,11 @@ Frequently Asked Questions
...
 
-- A: 
+- A: TBD
 
 **Q: What is community.mattermost kept on?**
  
-- A: For Feature releases community.mattermost is kept on the RCs. For Bug Fix releases keep it on master.
+- A: For Feature releases community.mattermost is kept on the RCs. For Bug fix releases keep it on master.
```
Comment (amyblais) [inline source/process/release-faq.rst]:
```suggestion
- A: For Feature releases community.mattermost is kept on the RCs. For Quality releases keep it on master.
```

File: source/process/release-faq.rst [hunk 4]
```diff
@@ -93,19 +93,19 @@ Frequently Asked Questions
...
+- A: Monitor the commit history of the respective release branch, e.g., https://github.com/mattermost/mattermost-server/commits/release-5.4 contains commits that shipped with mattermost-server v5.4. Jira ticket is resolved after cherry picking is done.
 
 **Q: What changes were made to the dev release process to account for the rotating feature and quality releases? https://developers.mattermost.com/internal/release-process/**
 
 - A: PR with changes was merged `here <https://github.com/mattermost/mattermost-developer-documentation/pull/182>`__.
 
 **Q: What changes were made to the team release process to account for the rotating feature and quality releases? https://docs.mattermost.com/process/release-process.html**
 
-- A: Separate checklists for `Bug Fix release <https://docs.mattermost.com/process/bug-fix-release.html>`__ and `Feature release <https://docs.mattermost.com/process/feature-release.html>`__ were created.
+- A: Separate checklists for `Bug fix release <https://docs.mattermost.com/process/bug-fix-release.html>`__ and `Feature release <https://docs.mattermost.com/process/feature-release.html>`__ were created.
```
Comment (amyblais) [inline source/process/release-faq.rst]:
```suggestion
- A: Separate checklists for `Quality release <https://docs.mattermost.com/process/bug-fix-release.html>`__ and `Feature release <https://docs.mattermost.com/process/feature-release.html>`__ were created.
```

File: source/process/release-faq.rst (hunk 1 above)
Comment (amyblais) [reply source/process/release-faq.rst]: This can be removed for now.

Comment (justinegeffen) [reply source/process/release-faq.rst]: Done.

File: source/process/release-faq.rst (hunk 2 above)
Comment (justinegeffen) [reply source/process/release-faq.rst]: Done. :)

**Pull request merged**